lua/lvim/bootstrap.lua

```diff
@@ -57,6 +57,11 @@ end
 ---Initialize the `&runtimepath` variables and prepare for startup
 ---@return table
 function M:init(base_dir)
+  -- for older installs. NVIM_APPNAME defaults to "lvim"
+  if not vim.env.NVIM_APPNAME then
+    vim.env.NVIM_APPNAME = "lvim"
+  end
```

Comment:
This will break the ability of using lvim's config as a neovim configuration, which used to be the best way to run on Windows.